telco-core/configuration/reference-crs-kube-compare/required/networking/Network.yaml

```diff
@@ -0,0 +1,12 @@
+# required
+# count: 1
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      gatewayConfig:
+        routingViaHost: true
+  useMultiNetworkPolicy: true
```

Comment:
Why are we enforcing useMultiNetworkPolicy for all customers?

Reply:
It should be optional.

Reply:
Its enabled in the current CRs in the repo. Which is what we based these templates on. 
https://github.com/openshift-kni/telco-reference/blob/main/telco-core/configuration/reference-crs/required/networking/Network.yaml#L25

But I'll make it optional if that's what it should be.